build.gradle

```diff
@@ -52,6 +52,12 @@ subprojects {
         }
     }
 
+    // Allow local appservices substitution in each subproject.
+    if (gradle.hasProperty('localProperties.autoPublish.application-services.dir')) {
+        ext.appServicesSrcDir = gradle."localProperties.autoPublish.application-services.dir"
+        apply from: "${rootProject.projectDir}/${appServicesSrcDir}/build-scripts/substitute-local-appservices.gradle"
```

Comment:
Hmmm, I feel like there's a better way to do this, but let's leave that for a follow-up.